src/main/java/org/jpeek/calculus/java/Ccm.java

```diff
@@ -68,6 +84,7 @@ public XML node(
      * @param skeleton XML Skeleton
      * @return XML with fixed NCC.
      */
+    @SuppressWarnings("PMD.UnusedPrivateMethod")
```

Comment:
really bed suppression, method is not used and confusing for all, why do we need it